addons/expanding-search-bar/expanding-search-bar.js

```diff
@@ -0,0 +1,72 @@
+export default async function ({ addon, global, console }) {
+
+	///scratch-www checking
+	var exsearch_isWWW; //True if scratch-www, false if not
+	if (document.getElementsByClassName("site-nav")[0]) {
+		exsearch_isWWW = false;
+	} else {
+		exsearch_isWWW = true;
+	}
```

Comment:
Don't we have an API for this????

Reply:
`addon.tab.clientVersion` should work, unless I'm misunderstanding what this does.